<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAssignmentWindow</class>
 <widget class="QWidget" name="NewAssignmentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Modalidade</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="modalityComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Região Anatômica</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="anatomicalRegionComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Descrição</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="descriptionText">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findFileButton">
       <property name="text">
        <string>Escolher imagens</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
